tests/pact/tc.repl

```diff
@@ -205,6 +205,29 @@
       (update persons "foo" partial)
       partial))
 
+  (defun fails-bind-bad-object:object{person} ()
+    (let ((o:object{person}
+            { 'name: 's
+            , 'age: 1
+            , 'dob: (parse-time "%F" "1996-12-31")
+            } ))
+      (bind o { 'name:= name, 'age:= age }
```

Comment:
How does this property (TC failure) is being checked? 
I assume this does not TC as the return object is lacking the `dob` field?

Reply:
See: `TypecheckSpec.hs`: https://github.com/kadena-io/pact/blob/64d58454d277394bfcbfc1882cdd2af343f8dd81/tests/TypecheckSpec.hs#L107

If particular expressions are named with `fails-`, as a prefix, we expect them to fail tc.

Reply:
Thank you, good to know!